src/test/resources/solr/dssolr/conf/schema.xml

```diff
@@ -90,7 +90,8 @@ In solrconfig.xml the classic schema is defined with this:  <schemaFactory class
     <?example     Aktuelt?>
   </field>
 
-  <field name="genre" type="string" docValues="true">
+  <!-- TODO: Extract main genre from preservica to this field and make single valued. Collect other genres in a new multivalued field-->
+  <field name="genre" type="string" docValues="true" multiValued="true">
```

Comment:
The `multiValued="true"` makes `EmbeddedSolrTest.testRecordUldallForTitleAndPlaceOfProductionAndGenre` fail.